moroScript.jsx

```diff
@@ -115,6 +117,23 @@
         return rtnWord;
     }
 
+    //Process dict with count to sorted dict without count value
+    function sortAndRemoveCount(dict) {
+        var toRtn = JSON.parse(JSON.stringify(dict));
+        for(var i=0; i<toRtn.length; ++i) {
+            toRtn[i]["moroword"].sort(function(a, b) {
+                return parseFloat(b["count"]) - parseFloat(a["count"]);
+            });
+            var moroWordsArray = []
+            for (var j=0; j<toRtn[i]["moroword"].length; ++j) {
```

Comment:
@maytasm: This looks great! Just F.Y.I, if you wanted to remove this for loop and just return toRtn without stripping off the "count" property, you could do that and just change the rendering code at:

https://github.com/sande570/Moro-database/blob/freqDict/moroScript.jsx#L231

For now I think I'll merge as is so that we can update http://sande570.github.com/Moro-database/, but I'll send you a PR to show you what that looks like if you are interesting.